### ISSUE TITLE
please add highlight and screenshot icons to the toolbar (or can be customized)

### ISSUE THREAD

**Issue by AHK0** [sumatrapdfreader/sumatrapdf#3324]
soooooooooo like this very nice software, thanks for your development.

could you please add highlight and screenshot icons in the toolbar, just like many other pdf viewers? or users can choose to show or hide those icons to the toolbar?

this is very helpful and time-saving for reading, and is easy to take screenshots of some regions (e.g., picture) in the pdf documents..
if possible, please offer several color options for the text highlight

**Comment by GitHubRulesOK** [comment]
Windows has the screenshot keys Win+Shift+S so no need for a different set
To select a region of SumatraPDF contents use Ctrl + Left mouse then that "Marque"  like acrobat can be zoom or copy or highlight with the A key thus no need for more icons as its key or mouse driven

There are 4,294,967,296 highlight colours for each type, but too many inefficient shades to pick from
simplest is you just set the stronger one of your choice for each type in the settings
#AaRdGnBl (AlphaRedGreenBlue as hex e.g. 0-F & 0-F) Not everyone wats the strongest but default Yellow would be #FFFFFF00

**Comment by AHK0** [comment]
> Windows has the screenshot keys Win+Shift+S (what I use here) so no need for a different set To select a region of SumatraPDF contents use Ctrl + Left mouse then that "Marque" like acrobat can be zoom or copy or highlight with the A key thus no need for more icons as its key or mouse driven ![image](https://user-images.githubusercontent.com/12754913/222902338-966e294d-0125-4bef-9992-50e6446d8dc4.png)
> 
> There are 4,294,967,296 highlight colours for each type, but too many inefficient shades to pick from simplest is you just set the stronger one of your choice for each type in the settings #AaRdGnBl (AlphaRedGreenBlue as hex e.g. 0-F & 0-F) Not everyone wants the strongest but default Yellow would be either #FFFFFF00 or #FFFF00 as shown above
> 
> ```
> Annotations [
> 	HighlightColor = #ffffff00
> 	UnderlineColor = #00ff00
> ```

great! already knew those functions. however, i would say that highlight and capture icons in the toolbar, instead of using keyboard, would make the software more popular for pc users.

**Comment by GitHubRulesOK** [comment]
say there are 1000 users (apparently over a million copies SOLD in china!!) and each wants just one or two icons, as have been constantly on different occasions requested then there would be little space for any text or pictures. and before you say floating that needs a programmer, so on occasion have added my own.

SumatraPDF adds a small single ribbon to the top edge of MuPDF which has NO icons only keyboard inputs, that ribbon as it stands is now wider than my smallest device but I can just about touch those important ones.

**Comment by AHK0** [comment]
you are right. unfortunately, nobody requests or forces you to add this or that function, and it does not matter where the users are. users only suggest some ideas, but it depends completely on whether the developers will follow. your logic seems to remind something: functions in other softwares or in the OS (screenshot, etc.) will not be developed for this software; there are many other similar software readers/viewers (pdf and other formats), so are you meaning to stop this sumatrapdf considering your logic?

let us stop/end this endless discussion. see what happens in years. some years ago, it was said that sumatrapdf will never have tabs, annotations, or others...

**Comment by kjk** [comment]
You can now customize toolbar: https://www.sumatrapdfreader.org/docs/Customize-toolbar

**Comment by GitHubRulesOK** [comment]
With 3.6.1+ Highlight color commands can be activated instantly via DDE so to make that easier I have updated the PicColo.HTA to apply colours instantly this version is PicColo.hta. 
https://github.com/GitHubRulesOK/SumatraPDF-Plus/blob/master/PicColo.hta
You select a text press a key for the colour bar and on mouse click the colour is applied. (you could easily edit it to not close on use)
```
ExternalViewers [
	[
		CommandLine = "C:\Users\WDAGUtilityAccount\Desktop\Apps\PDF\SumatraPDF\17629\PicColo.hta"
		Name = PicColo (Pick a &HL colour)
		Filter = *.pdf
		Key = h
	]
]
```
<img width="803" height="393" alt="Image" src="https://github.com/user-attachments/assets/28a407d6-c53e-4478-8bd8-f3c88114a13b" />

<img width="801" height="406" alt="Image" src="https://github.com/user-attachments/assets/4ba12b21-ff84-48f5-b804-b0738966a58b" />

**Comment by GitHubRulesOK** [comment]
@kjk withe addition of new  printscreen functions that can be set as text Icon I am closing as all OP requirements now met 

<img width="847" height="96" alt="Image" src="https://github.com/user-attachments/assets/41afeadc-ca4a-4746-a947-8f3db369b197" />
